release.py

```diff
@@ -256,11 +260,105 @@ def command_stage_docs():
 
     sys.exit(0)
 
+def command_release_announcement_email():
+    release_version_num = raw_input('What is the current release version:')
+    previos_release_version_num = raw_input('What is the previous release version:')
+    number_of_contributors = int(subprocess.check_output('git shortlog -sn --no-merges %s..%s | wc -l' % (previos_release_version_num, release_version_num) , shell=True))
+    contributors = subprocess.check_output("git shortlog -sn --no-merges %s..%s | cut -f2 | tr '\n' ',' | sed -e 's/,/, /g'" % (previos_release_version_num, release_version_num), shell=True)
+    release_announcement_data = {
+        'number_of_contributors': number_of_contributors,
+        'contributors': contributors,
+        'release_version': release_version_num
+    }
+
+    release_announcement_email = """
+        To: dev@kafka.apache.org, users@kafka.apache.org, kafka-clients@googlegroups.com
+        Subject: [ANNOUNCE] Apache Kafka %(release_version)s
+        
+        The Apache Kafka community is pleased to announce the release for Apache Kafka %(release_version)s
+        
+        <DETAILS OF THE CHANGES>
+        
+        All of the changes in this release can be found in the release notes:
+        https://www.apache.org/dist/kafka/%(release_version)s/RELEASE_NOTES.html 
+ 
+
+        You can download the source and binary release (Scala 2.11 and Scala 2.12) from:        
```

Comment:
Not sure how critical it is that we automate this, but these are configurable in the build now (see `availableScalaVersions`/`defaultScalaVersions` in `dependencies.gradle`). I'm a bit wary of trying to automate just because it requires parsing gradle code to do robustly, but at the same time, this seems like it'd be easy to have fall out of date.

I guess we could leave as is and hopefully people will be comprehensively `git grep`ing for places they need to change when adding/removing scala versions?

Reply:
I double people will update this script correctly. We can only hope, that release managers verify this before sending the email...

As an alternative, we can also wildcard this, and let release manger insert those manually. Similar to `<DETAILS OF THE CHANGES>` above.